### PR TITLE
Enhance add message stack

### PR DESCRIPTION
**Pull request by mafoo**
Support legacy $_SESSION['message_*']
Use new messages class
New function messages::add($message, $mood, $delay) to simplify adding messages
- [x] Tested on debian 8, Host firefox client

simplification of giving messages, allows multiple messages to be displayed with individual moods and delays

### CODE REVIEW

File: resources/functions.php [hunk 1]
```diff
@@ -1917,4 +1917,10 @@ function event_socket_mkdir($dir) {
 			return false;
 	}
 
+	if (!function_exists('add_message')) {
+		function add_message($message, $mood, $delay) {
```
Comment (moteus) [inline resources/functions.php]: I think need provide default values in parameter list to php 7

**Comment by markjcrane** [comment]
I wondering a bit before effort is used to place across fusionpbx whether it would be better to use an object with add method instead of function add_message. Thoughts on this?

File: resources/functions.php (hunk 1 above)
Comment (markjcrane) [inline resources/functions.php]: PHP 7.1 will error if one of the function params are not set with a default value and not provided to the function when its called.

**Comment by markjcrane** [comment]
A class is more flexible then functions alone.

**Comment by mafoo** [comment]
Class has been implemented as requested

**Comment by markjcrane** [comment]
That looks great! Separates the array data from the display much better this way similar concept to how the menu is handled. +1

**Pull request merged**